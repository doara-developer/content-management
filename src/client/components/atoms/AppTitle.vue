<template>
    <a href="/">
        <h1 class="app-title"><slot /></h1>
    </a>
</template>
<script lang="ts">
import Vue from "vue";

export default Vue.extend({});
</script>
<style lang="scss" scoped>
.app-title {
    color: $header-font-color;
}
</style>
