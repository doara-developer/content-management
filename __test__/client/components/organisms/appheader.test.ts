import { shallowMount } from "@vue/test-utils";
import AppTitle from "@client/components/atoms/AppTitle.vue";

describe("AppTitle.vue", () => {
    test("renders props.msg when passed", () => {
        const wrapper = shallowMount(AppTitle);
        expect(wrapper).toMatchSnapshot();
    });
});
