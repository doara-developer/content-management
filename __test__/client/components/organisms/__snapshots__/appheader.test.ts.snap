// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTitle.vue renders props.msg when passed 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
